gramps/gui/grampsgui.py

```diff
@@ -490,7 +491,7 @@ def _display_welcome_message(parent=None):
     Display a welcome message to the user.
     (This docstring seems very legacy/historical, not accurate.)
     """
-    _display_generic_message("master", 'behavior.betawarn', parent=parent)
+    _display_generic_message(VERSION, 'behavior.betawarn', parent=parent)
```

Comment:
It's probably better to put an `if DEV_VERSION:` here, rather than modifying the `_display_generic_message` function.

Reply:
I expect to update the release procedure anyway, because it never mentioned these files, thus the issue.

I considered the original simple flag, but if it ever got set, the user would never see a warning again for any other development version.  With the tuple, he will get a new warning every time he runs a different development version.

I actually considered simplifying one of the _display_generic_message or _display_welcome_message methods out of existence, since only one bit of code ever calls the latter, and only it calls the former.  And it is not very 'generic'.  Original change to create the generic method courtesy of Paul Franklin.

Reply:
I would prefer an `if` statement around the lines that we previously commented out.